deps/chakrashim/src/jsrtcontextshim.cc

```diff
@@ -368,11 +368,34 @@ bool ContextShim::InitializeProxyOfGlobal() {
   });
 }
 
-bool ContextShim::EnsureInitialized() {
+
+bool ContextShim::InitializeCurrentContextShim() {
```

Comment:
Clarify... "EnsureInitialized" is more like a public method name. You can still split the task to 2 parts like current, and call the other one just "Initialize" or "DoInitialize", which is private and only called by "EnsureInitialized". All usages only call "EnsureInitialized", a bit more clear.